testcases/broker-kc-config/100-broker.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-kafkachannel-broker
+data:
+  channelTemplateSpec: |-
+    apiVersion: messaging.knative.dev/v1beta1
+    kind: KafkaChannel
+    spec:
+        numPartitions: 1
+        replicationFactor: 1
+---
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: my-kafkachannel-broker
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: my-kafkachannel-broker
```

Comment:
@lberk I am using the `ConfigMap` based approach here, instead of overriding some system defaults.

Let me know what you think :smile_cat:

Reply:
:+1:

Reply:
ya I think that makes, instead of messing with defaults, set up test specific settings, +1 from me